deeprankcore/trainer.py

```diff
@@ -519,6 +519,7 @@ def train( # pylint: disable=too-many-arguments, too-many-branches, too-many-loc
         shuffle: bool = True,
         earlystop_patience: Optional[int] = None,
         earlystop_maxgap: Optional[float] = None,
+        min_epoch: int = 10, 
```

Comment:
I think it would be nice to also make this an Optional. I realize that would be the same as setting it to 0, but I think setting it as None is more explicit that you don't want to use a minimum epoch.

```suggestion
        min_epoch: Optional[int] = 10, 
```